### PR TITLE
Add new extended format to scan doc - with suggestions examples and s…

### PR DESCRIPTION
**Pull request by dmartin35**
…ummary recap

Signed-off-by: David Martin <david@chaosiq.io>

### CODE REVIEW

File: docs/getting-started/scan-your-first-project/index.md [hunk 1]
```diff
@@ -76,3 +76,35 @@ resource containing the concerning line.
   contained another resource, say a service, before this deployment the
   line number would be different.
 :::
+
+
+### Show suggestion with best practice example
```
Comment (mperrien) [inline docs/getting-started/scan-your-first-project/index.md:81]: I did not find this very clear about what the section was about. Is "best practice example" the definitive name for this feature?
I'm wondering if something around "resolution" would work.
Something like "Show suggestions with resolution hints"

Any ideas @Lawouach?

File: docs/getting-started/scan-your-first-project/index.md [hunk 2]
```diff
@@ -76,3 +76,35 @@ resource containing the concerning line.
...
+
+When running the CLI scanning, the resulting suggestions can be displayed
```
Comment (mperrien) [inline docs/getting-started/scan-your-first-project/index.md:83]:
```suggestion
When running a CLI scan, the resulting suggestions can be displayed
```

File: docs/getting-started/scan-your-first-project/index.md [hunk 3]
```diff
@@ -76,3 +76,35 @@ resource containing the concerning line.
...
+with an optional example of best practice to be applied. To do so, you'll
```
Comment (mperrien) [inline docs/getting-started/scan-your-first-project/index.md:84]:
```suggestion
with an optional example of a best practice to be applied. To do so, you'll
```
I think the "a" is mandatory here for singular.

File: docs/getting-started/scan-your-first-project/index.md [hunk 4]
```diff
@@ -76,3 +76,35 @@ resource containing the concerning line.
...
+need to run the `scan` command with the specific `extended` format:
+
+```console
+$ reliably scan . -f extended
+```
+
+You will get as output the list of suggestions with examples, as well as a
```
Comment (mperrien) [inline docs/getting-started/scan-your-first-project/index.md:91]:
```suggestion
The CLI will output the list of suggestions with examples, as well as a
```

**Review by mperrien — COMMENTED** [review]
Minor changes,  but I'm wondering if "best practice" is the right phrase here. For example, I didn't understand the title of the section at first. I thought you were going to explain best practices for using the CLI. I mentioned @Lawouach in the specific comment to see if he had ideas.

File: docs/getting-started/scan-your-first-project/index.md (hunk 1 above)
Comment (mperrien) [reply docs/getting-started/scan-your-first-project/index.md:81]: Good for me!

**Comment by dmartin35** [comment]
The feature is simply to be able to show an example for each suggestion being returned

**Comment by Lawouach** [comment]
It all looks good to me.

**Pull request merged**